todo_app/app.py

```diff
@@ -1,40 +1,96 @@
+    #Select the database   
+    db = client.todo_app    
+    #Select the collection
+    collection = db.todolist
     app = Flask(__name__)
     app.config.from_object(Config())
     
-    init_trello()
-
+    
+    def todolists():
+        dicttodo=collection.find({"Status" : "To Do"})
+        todos = []
+        for todo_item in dicttodo:
+            todos.append(Item.from_mongo(todo_item))
+        return todos
+        
+    def doinglists():
+            dictdoing=collection.find({"Status" : "In Progress"})
+            doing = []
+            for doing_item in dictdoing:
+                doing.append(Item.from_mongo(doing_item))
+            return doing
+            
+    def donelists():
+            dictdone=collection.find({"Status" : "Completed"})
+            done = []
+            for done_item in dictdone:
+                done.append(Item.from_mongo(done_item))
+            return done
+    
     @app.route('/')
     def index():
         todo_items = todolists()
         doing_items = doinglists()
-        done_items = donelists()
-        item_view_model = ViewModel(todo_items, doing_items, done_items)
-        #return render_template('index.html', view_model=item_view_model)
+        done_item = donelists()
+        item_view_model = ViewModel(todo_items, doing_items, done_item)
         return render_template('index.html', view_model = item_view_model)
-
+    
     @app.route('/additem', methods = ["POST"])
     def add_new_item():
-        new_card()
-        return redirect('/')
+        name=request.values.get("name")
+        desc=request.values.get("desc")
+        collection.insert_one({ "name":name, "desc":desc, "Status": "To Do"})
+        return redirect('/')   
 
     @app.route('/movedoing', methods = ["POST"])
     def movedoing():
-        move_doing()
-        return redirect('/')
+	    id=request.values.get("_id")
+	    task=collection.find({"_id":ObjectId(id)})
+	    if(task[0]["Status"]=="To Do"):
+		    collection.update_one({"_id":ObjectId(id)}, {"$set": {"Status":"In Progress"}})
+	    else:
+		    collection.update_one({"_id":ObjectId(id)}, {"$set": {"Status":"Completed"}})
+	    return redirect('/')
+    
 
     @app.route('/movedone', methods = ["POST"])
     def movedone():
-        move_done()
-        return redirect('/')
+	    id=request.values.get("_id")
+	    task=collection.find({"_id":ObjectId(id)})
+	    if(task[0]["Status"]=="In Progress"):
+		    collection.update_one({"_id":ObjectId(id)}, {"$set": {"Status":"Completed"}})
+	    else:
+		    collection.update_one({"_id":ObjectId(id)}, {"$set": {"Status":"In Progress"}})
+	    return redirect('/')
+
+    @app.route('/delete', methods = ["POST"])
+    def remove():
+	    id=request.values.get("_id")
+	    task=collection.find({"_id":ObjectId(id)})
+	    if(task[0]["Status"]=="Completed"):
+		    collection.delete_one({"_id":ObjectId(id)})
+	    else:
+		    collection.delete_one({"_id":ObjectId(id)})
```

Comment:
This has the structure:
```
If <x>:
  Delete the element
Else:
  Delete the element 
```
Which seems to simplify to just `Delete the element`!

```suggestion
	    collection.delete_one({"_id":ObjectId(id)})
```